geoips/plugins/yaml/sectors/static/goes17.yaml

```diff
@@ -1,22 +1,26 @@
-goes17:
+interface: sectors
+family: area_definition
+name: goes17
+docstring: "Full disk sector for GOES-17"
+metadata:
+  continent: Global
+  country: x
+  area: GOES-17
+  subarea: x
+  state: x
+  city: x
+spec:
+  area_id: goes17
   description: GOES-17 Coverage
   projection:
     a: 6371228.0
     lat_0: 0
-    lon_0: -137.2
```

Comment:
Was this intentionally changed?

Reply:
No, this was accidental. I just changed it back.